tests/utils.py

```diff
@@ -11,29 +11,13 @@
 from httpx import AsyncClient
 from sqlalchemy import Connection as AlchConnection
 from sqlalchemy import MetaData, pool, text
-from sqlalchemy.ext.asyncio import (
-    AsyncConnection,
-    async_engine_from_config,
-    create_async_engine,
-)
+from sqlalchemy.ext.asyncio import AsyncConnection, async_engine_from_config
 
-from syncmaster.config import Settings
 from syncmaster.db.models import Status
 
 logger = logging.getLogger(__name__)
 
 
-async def prepare_new_database(settings: Settings) -> None:
```

Comment:
This is important for migration tests

Reply:
for now migrations anyway run in main database` syncmaster` and previous test database `postgres` is not used anywhere, with @IlyasDevelopment decided to implement test database in separate task

Reply:
`test_migrations_up_to_date` should use a separated database to apply migrations from the start to the end. Running these migrations in main database may be inconvenient for local development, as it may contain some test data which will be erased.